Sealnote/src/main/java/com/twistedplane/sealnote/NoteActivity.java

```diff
@@ -107,6 +107,12 @@ public void afterTextChanged(Editable editable) {
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_note);
+
+        if(!Misc.isPasswordLoaded()){
```

Comment:
(1) Please add a whitespace between "if" and the parenthesis.

https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace